src/utils/helpers/ReducerHelper.jsx

```diff
@@ -15,3 +15,29 @@ export const completeTask = (todos, payload) => {
   });
   return newList;
 };
+
+export const editTask = (todos, task) => {
+  const newList = todos.map((todo) => {
+    if (todo.id === task.taskId) {
+      return {
+        ...todo,
+        taskDetails: task.taskDetails
+      };
+    }
+    return todo;
+  });
+  return newList;
+};
+
```

Comment:
Are we rendering any XML here? If not, shouldn't the file name extension be *.js instead of *.jsx?

Reply:
file renamed